travis-build/Makefile

```diff
@@ -21,11 +23,12 @@ ${ROCKSDB_DIR}/librocksdb.so: ${ROCKSDB_DIR}/libgflags.a
 	make shared_lib && \
 	cp librocksdb.so* ${ROCKSDB_DIR}
 
+# TODO: use offical archive once https://github.com/SimonKagstrom/kcov/pull/146 is resolved and published.
```

Comment:
No, `use official archive` means use a stable version of official archive. So we should use a specific version or a version with a patch, latter is my very branch.